lib/coffee-linter.js

```diff
@@ -12,6 +14,7 @@ function CoffeeScriptLinter (inputTree, options) {
   if (!(this instanceof CoffeeScriptLinter)) return new CoffeeScriptLinter(inputTree, options);
   Filter.call(this, inputTree, options);
   options = options || {};
+  this.options = options;
```

Comment:
Could you set the `options.persist = true`, and then move the `Filter.call` line to after this?